src/f32-qs8-vcvt/rvv.c.in

```diff
@@ -0,0 +1,55 @@
+// Copyright 2024 Imagination Technologies, inc.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+$assert LMUL in [1, 2, 4, 8]
+$LMUL_8 =  {1: "f4", 2: "f2", 4: "1", 8: "2"}[LMUL]
+$LMUL_16 = {1: "f2", 2: "1",  4: "2", 8: "4"}[LMUL]
+$XINT8_T = {"QS8": "int8_t", "QU8": "uint8_t"}[DATATYPE]
+#include <assert.h>
+
+#include <riscv_vector.h>
+#include "xnnpack/math.h"
+#include "xnnpack/vcvt.h"
+
+
+void xnn_f32_${DATATYPE.lower()}_vcvt_ukernel__rvv_u${LMUL}v(
+    size_t batch,
+    const float* input,
+    ${XINT8_T}* output,
+    const union xnn_f32_${DATATYPE.lower()}_cvt_params params[restrict XNN_MIN_ELEMENTS(1)]) XNN_OOB_READS
```

Comment:
RVV never has OOB_READS?   this tells asan not to check reads, but we have vsetvl that should ensure reads stay in bounds

Reply:
Other RVV kernels had this enabled by default so I didn't investigate further into this. Would we want to be consistent with other kernels, or remove this?